### PR TITLE
Fix ESLint errors with new syntax & style rules

### PR DESCRIPTION
**Pull request by omh1280**
Bump the three `package.json` files to require `eslint-config-cesium: ^2.0.0` and fix all errors with the new rules.

For https://github.com/AnalyticalGraphicsInc/cesium/issues/5456.

Fix all the errors ESLint throws with the new 2.0 additions. Includes [global-require](http://eslint.org/docs/rules/global-require), [no-buffer-constructor](http://eslint.org/docs/rules/no-buffer-constructor), and [no-new-require](http://eslint.org/docs/rules/no-new-require) rules.

### CODE REVIEW

**Comment by omh1280** [comment]
I'll switch the ESLint task to use CLI soon

**Comment by mramato** [comment]
Thanks, just bump when ready and I'll review/merge.

**Comment by omh1280** [comment]
Ready for a review when you get the chance!

File: generator/.gitignore [hunk 1]
```diff
@@ -7,6 +7,7 @@ npm-debug.log
 .idea/tasks.xml
 
 # Generated files
+.eslintcache
```
Comment (mramato) [inline generator/.gitignore]: Add this to `.npmignore` as well (all of the ones in this repository).

**Comment by mramato** [comment]
Just that one comment and I'll merge.  Thanks @omh1280!

**Comment by omh1280** [comment]
Done!

**Comment by mramato** [comment]
@omh1280 can you merge in master.  There are a bunch of (hopefully minor) conflicts.  Thanks.

**Comment by omh1280** [comment]
Ok! I think I merged everything correctly.

**Comment by mramato** [comment]
Looks good!  Thanks.

**Pull request merged**